test/markup/cpp/function-title.txt

```diff
@@ -1,5 +1,5 @@
-int main() {
-  A a = new A();
+int main(int argc, char** argv) {
+  auto a = std::make_unique<A>();
```

Comment:
Please add a second test (or just a second file) in this file vs *removing* the simple case to replace it with a complex case.

```suggestion
  A a = new A();
  auto a = std::make_unique<A>();
```

Reply:
Yes, will do.

I changed this, because I got autodetection issues with the old version; it was detected as PHP for some reason.

Reply:
Markup (`tests/markup`) tests have _nothing_ to do with auto-detect tests.

Reply:
Yeah I'm sorry, I think I was confused die to the outcome of the auto detection in the developer markup tool.